Gopkg.toml

```diff
@@ -0,0 +1,34 @@
+
+# Gopkg.toml example
```

Comment:
hmm we shouldn't need this file anymore with go modules, I wonder how it got created :thinking: Did you run ``dep init`` or was it the IDE?

Reply:
I guess it lingered from when it was needed... not sure, will remove.